lib/screens/landing_page/landing_page.dart

```diff
@@ -0,0 +1,200 @@
+import 'package:flutter/material.dart';
+import 'package:promdi_fe/helpers/style.dart';
+import 'package:promdi_fe/screens/login/login.dart';
+import 'package:promdi_fe/screens/signup/signup_page.dart';
+import 'package:promdi_fe/widgets/card.dart';
+
+class LandingPage extends StatefulWidget {
+  const LandingPage({Key? key}) : super(key: key);
+
+  @override
+  _LandingPageState createState() => _LandingPageState();
+}
+
+class _LandingPageState extends State<LandingPage> {
+  DateTime currentDate = DateTime.now();
+  @override
+  Widget build(BuildContext context) {
+    final height = MediaQuery.of(context).size.height;
+    return Scaffold(
+      appBar: AppBar(
+        centerTitle: true,
+        title: Text('Promdi App'),
+        leading: ClipOval(
+          child: Image.asset(
+            'assets/icons/logo.png',
+            fit: BoxFit.cover,
+            height: height * 0.04,
+          ),
+        ),
+      ),
+      body: Container(
+        height: height,
+        color: lightGrey,
+        child: SingleChildScrollView(
+          child: Column(
+            children: [
+              Card(
+                color: lightBlue,
+                child: Padding(
+                  padding: EdgeInsets.all(10),
+                  child: Row(
+                    mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                    children: [
+                      Image.asset(
+                        'assets/images/cloudy.png',
+                        height: height * 0.1,
+                      ),
+                      Column(
+                        children: [
+                          Text(
+                            '25c',
+                            style: TextStyle(
+                                color: light,
+                                fontWeight: FontWeight.bold,
+                                fontSize: 20),
+                          ),
+                          SizedBox(height: 5),
+                          Text(
+                              '${currentDate.day} - ${currentDate.month} - ${currentDate.year}',
```

Comment:
I'm requesting we put a date such as Aug 10th, 2021

Reply:
okay